frontend/src/components/object/ObjectList.vue

```diff
@@ -54,62 +54,76 @@ const listObjects = async () => {
   }
 };
 
+// const updateBreadcrumb = async () => {
```

Comment:
Same here - wondering if we need to keep the commented code?

Reply:
For this one, we want to revisit breadcrumbs at a later date, so this should be preserved.